velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -991,6 +991,26 @@ TEST_P(AsyncDataCacheTest, cacheStats) {
   ASSERT_EQ(cache_->toString(false), expectedShortCacheOutput);
 }
 
+TEST_P(AsyncDataCacheTest, cacheStatsWithSsd) {
```

Comment:
Can we also add cases other than the initial case to make the test complete? Thanks